packages/grafana-ui/src/components/Combobox/Combobox.story.tsx

```diff
@@ -51,17 +52,19 @@ const meta: Meta<PropsAndCustomArgs> = {
 const BasicWithState: StoryFn<typeof Combobox> = (args) => {
   const [value, setValue] = useState(args.value);
   return (
-    <Field label="Test input" description="Input with a few options">
-      <Combobox
-        id="test-combobox"
-        {...args}
-        value={value}
-        onChange={(val) => {
-          setValue(val?.value || null);
-          action('onChange')(val);
-        }}
-      />
-    </Field>
+    <Box maxWidth={80}>
```

Comment:
This is my suggestion for setting the maxWidth, instead of having the prop on Combobox. Not sure how we should approach minWidth though.

Reply:
Because Combobox has a auto-sizing prop (whatever that actual API is), it should also have props to control that auto-sizing behaviour. I think `minWidth` and `maxWidth` should be added to Combobox to match the auto-sizing prop.

Additionally, I think `minWidth` should be mandatory when auto-sizing, to prevent impractically narrow inputs by default.